.vscode/settings.json

```diff
@@ -6,5 +6,6 @@
   "[typescript]": {
     "editor.formatOnSave": false
   },
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "editor.wordWrapColumn": 100
```

Comment:
Can you please remove this line (wordWrapColumn)

Reply:
If this is a problem, then we should consider removing settings.json from the repo

Reply:
I don't thin it would be